### PR TITLE
S3 t6 taskbar functionality

### PR DESCRIPTION
**Pull request by SlothSan**

### CODE REVIEW

File: src/Component/Applications/AboutApplication/SubComponents/Header/Header.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .handle {
-  width: 100%;
+  width: 101%;
```
Comment (JVLoe) [inline src/Component/Applications/AboutApplication/SubComponents/Header/Header.scss:2]: why 101%?

File: src/Component/Applications/AboutApplication/AboutApplicationContent/AboutApplicationContent.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+                Etiam vitae sapien quis metus mattis commodo eget eget ipsum. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aenean neque tortor, feugiat et dui id, efficitur hendrerit ex. Nulla bibendum tellus id ligula tincidunt, ut hendrerit purus ullamcorper. Pellentesque id dignissim ante, in efficitur dui. Nunc maximus vel quam ut euismod. In a metus vestibulum, aliquam dolor id, pharetra nulla. Ut tincidunt id lorem sed tristique.
+
+                Etiam vitae sapien quis metus mattis commodo eget eget ipsum. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aenean neque tortor, feugiat et dui id, efficitur hendrerit ex. Nulla bibendum tellus id ligula tincidunt, ut hendrerit purus ullamcorper. Pellentesque id dignissim ante, in efficitur dui. Nunc maximus vel quam ut euismod. In a metus vestibulum, aliquam dolor id, pharetra nulla. Ut tincidunt id lorem sed tristique.
```
Comment (JVLoe) [inline src/Component/Applications/AboutApplication/AboutApplicationContent/AboutApplicationContent.tsx]: Should we do this before merge?

File: src/Component/TaskBar/StartButton/index.tsx [hunk 3]
```diff
@@ -12,15 +14,19 @@ const StartButton = (props :StartButtonProps): JSX.Element => {
...
 
     const showMenu = () => {
-        setToggle(!toggle);
-        if (toggle === true) {
+        props.setStartMenuOpen(!props.startMenuOpen);
+    }
+
+    useEffect(() => {
+        if (!props.startMenuOpen) {
             // @ts-ignore
```
Comment (JVLoe) [inline src/Component/TaskBar/StartButton/index.tsx:22]: leave the comments for now and delete before main merge?

File: src/Component/TaskBar/StartButton/index.tsx [hunk 4]
```diff
@@ -12,15 +14,19 @@ const StartButton = (props :StartButtonProps): JSX.Element => {
...
-           document.getElementById( "menu").style.display="none";
-        } else {
-           // @ts-ignore
             setWidth("block");document.getElementById("menu").style.display="block";
+
```
Comment (JVLoe) [inline src/Component/TaskBar/StartButton/index.tsx]: delete

File: src/Component/TaskBar/TaskDock/TaskDock.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import './TaskDock.css'
```
Comment (JVLoe) [inline src/Component/TaskBar/TaskDock/TaskDock.tsx:1]: TaskDock? what is this referring to? semantically-wise

File: src/Component/TaskBar/TaskDock/TaskDock.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+interface TaskDockProps {
+    setAboutIsOpen: Function
+    setAboutIsRunning: Function
+    aboutIsOpen: boolean
+    aboutIsRunning: boolean
+}
+
```
Comment (JVLoe) [inline src/Component/TaskBar/TaskDock/TaskDock.tsx:9]: delete

File: src/Component/TaskBar/TaskDock/TaskDock.tsx (hunk 5 above)
Comment (JVLoe) [reply src/Component/TaskBar/TaskDock/TaskDock.tsx:1]: oh wait, this is the taskbar?

File: src/Component/Applications/AboutApplication/SubComponents/Header/Header.scss (hunk 1 above)
Comment (SlothSan) [reply src/Component/Applications/AboutApplication/SubComponents/Header/Header.scss:2]: Talked about in person.

File: src/Component/Applications/AboutApplication/AboutApplicationContent/AboutApplicationContent.tsx (hunk 2 above)
Comment (SlothSan) [reply src/Component/Applications/AboutApplication/AboutApplicationContent/AboutApplicationContent.tsx]: Talked in person.

File: src/Component/TaskBar/StartButton/index.tsx (hunk 4 above)
Comment (SlothSan) [reply src/Component/TaskBar/StartButton/index.tsx]: Removed

File: src/Component/TaskBar/TaskDock/TaskDock.tsx (hunk 5 above)
Comment (SlothSan) [reply src/Component/TaskBar/TaskDock/TaskDock.tsx:1]: It is humorful name for a taskbar.

File: src/Component/TaskBar/TaskDock/TaskDock.tsx (hunk 6 above)
Comment (SlothSan) [reply src/Component/TaskBar/TaskDock/TaskDock.tsx:9]: Resolved

**Pull request merged**